manifests/testing/cdi-v1.4.1.yaml.in

```diff
@@ -47,7 +50,6 @@ apiVersion: rbac.authorization.k8s.io/v1beta1
 kind: ClusterRoleBinding
 metadata:
   name: cdi-sa
-  namespace: {{.CDINamespace}}
```

Comment:
should this be removed?

Reply:
ClusterRole and ClusterRoleBinding aren't namespaced.